tests/test_core.py

```diff
@@ -59,29 +59,49 @@ def test_check_response_ok(self, status_code, content):
         response = Response(status_code, content=content)
         check_response(response)
 
-    def test_check_response_bad_stream(self, caplog):
+    @pytest.mark.parametrize('log_errors', (True, False))
+    def test_check_response_bad_stream(self, caplog, log_errors):
         iterable = [b'iter', b'lines']
         status_code = httplib.CONFLICT
         response = Response(status_code, iterable=iterable)
+        if log_errors:
+            log_type = logging.ERROR  # 40
+        else:
+            log_type = logging.DEBUG  # 10
```

Comment:
Not sure if these comments add anything. We don't care what the values of these are do we?